Core/Core/Api/Exceptions.cs

```diff
@@ -13,8 +13,8 @@ namespace Speckle.Core.Api;
 /// </summary>
 public class SpeckleGraphQLException<T> : SpeckleException
 {
-  private GraphQLRequest _request;
-  public readonly GraphQLResponse<T>? Response;
+  private readonly GraphQLRequest _request;
+  public GraphQLResponse<T>? Response { get; }
```

Comment:
Shouldn't the Response also be `readonly`?

Reply:
Response was previously a public readonly field.
I've made it a get only property, since public fields should be avoided.
So in that sense, by being a `{ get; }` its already read-only, you can't use the `readonly` keyword with class properties.

Reply:
oh yes... obviously 🤦🏼‍♂️